internal/app/handlers/post_handler.go

```diff
@@ -0,0 +1,29 @@
+package handlers
+
+import (
+	"mishin-shortener/internal/app/hasher"
+	"mishin-shortener/internal/app/storage"
+	"io"
+	"net/http"
+)
+
+func PostHandler(w http.ResponseWriter, r *http.Request, db *storage.Database, redirectHost string) {
+	if r.RequestURI == "/" {
+
+		body, err := io.ReadAll(r.Body)
```

Comment:
Стоит закрыть r.Body, при завершении работы хендлера.
```suggestion
		body, err := io.ReadAll(r.Body)
		defer r.Body.Close()
```

Reply:
Добавил, но в теле курса видел, что можно не закрывать и что оно закрывается само.

Reply:
Действительно! Спасибо, что указал на это 🙏